src/error-snippets/index.ts

```diff
@@ -14,7 +14,7 @@ const stackFrameLocationResolver = async (stackFrame: SufficientStackFrame): Pro
 };
 
 export const extendWithCodeSnippet = async (err: WithSnippetError): Promise<WithSnippetError> => {
-    if (!err) {
+    if (!err || shouldNotAddCodeSnippet(err)) {
```

Comment:
we can pass error to the `shouldNotAddCodeSnippet` function and then check all cases inside of the function